src/ethereum/ethEngine.js

```diff
@@ -724,10 +725,9 @@ export class EthereumEngine extends CurrencyEngine {
     }
 
     // Select the chain
-    const common = new Common({
-      // EIP 155 chainId - ETH mainnet: 1, ETH ropsten: 3
-      chain: this.currencyInfo.defaultSettings.otherSettings.chainId
-    })
+    const { chainParams } = (this.currencyInfo.defaultSettings
+      .otherSettings: EthereumSettings)
```

Comment:
The colon syntax breaks VSCode pretty bad, so we typically do something like this when we need to do a cast:

```js
const otherSettings: EthereumSettings = this.currencyInfo.defaultSettings.otherSettings
const { chainParams } = otherSettings
```